src/Products/List/index.jsx

```diff
@@ -1,22 +1,70 @@
-import React from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
-
+import { Row, Col } from 'react-flexbox-grid';
+import get from '../../api';
+import Card from './Card';
 import TopPanel from './TopPanel';
-import ItemGrid from './ItemGrid';
+import priceTransform from '../../Components/Price';
+import getImageLink from '../../Components/ImageLink';
 
 const Wrapper = styled.main`
   width: 100%;
   @media screen and (min-width: 768px){
-      width: calc(100% - 327px);
+      width: 66%;
   }
   @media screen and (min-width: 992px){
-      width: calc(100% - 414px);
+      min-width: calc(100% - 414px);
   }
 `;
 
-export default () => (
-  <Wrapper>
-    <TopPanel />
-    <ItemGrid />
-  </Wrapper>
-);
+const GridWrapper = styled.div`
+  box-sizing: border-box;
+  margin: 18px 15px;
+  @media screen and (min-width: 768px){
+    margin: 24px 25px;
+  }
+`;
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: [],
+    };
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData(this.props);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.fetchData(nextProps);
+  }
+
+  fetchData(props) {
+    get(`v1/${props.match.url}`).then(json => this.setState({ products: json.items }));
```

Comment:
`.then(({ items: products }) => this.setState({ products })`

object destruct + parameter context matching + object shorthand